File: dataset/avatar/Python/Code/codeforces_653_A.py
n = int ( input ( ) )
l = list ( map ( int , input ( ).split ( ) ) )
l.sort ( )
for i in l :
    if i + 1 in l and i + 2 in l :
        print ( 'YES' )
        break
else :
    print ( 'NO' )

File: output/deepseek_chat/avatar/algo_based_translation/python/java/successful_translation/codeforces_653_A.java

import java.util.*;

public class codeforces_653_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        List<Integer> l = new ArrayList<>();
        String[] input = scanner.nextLine().split(" ");
        for (String s : input) {
            l.add(Integer.parseInt(s));
        }
        Collections.sort(l);
        boolean found = false;
        for (int i : l) {
            if (l.contains(i + 1) && l.contains(i + 2)) {
                System.out.println("YES");
                found = true;
                break;
            }
        }
        if (!found) {
            System.out.println("NO");
        }
    }
}

